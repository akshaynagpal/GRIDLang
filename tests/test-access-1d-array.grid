int gameloop(){
    int j;
    int[4] x;
    x = [1,2,3,4];
    x[0]=5;
    j = x[0];
    print(j);
    return 0;
}

int checkGameEnd()
{
  return 1;
}