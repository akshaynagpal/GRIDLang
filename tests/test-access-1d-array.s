	.text
	.file	"./tests/test-access-1d-array.ll"

	.section	".note.GNU-stack","",@progbits
