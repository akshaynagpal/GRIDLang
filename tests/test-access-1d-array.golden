Success. Able to access a value at index of 1d array.